### PR TITLE
Takes away sci comms from miners for tipy because they're too admin to code so they finally shut up

### PR DESCRIPTION
**Pull request by alexkar598**
<details>
<summary>Click here for nerd view</summary>

### Intent of your Pull Request
Takes away sci comms from miners

### Why is this change good for the game?
Idk ask tipy

### Briefly describe your PR and the impacts of it, in layman's terms. 
Takes away sci comms from miners

### What should players be aware of when it comes to the changes your PR is implementing?
Takes away sci comms from miners

### What general grouping does this PR fall under? 
Taking away sci comms from miners

:cl:  
rscdel: Takes away sci comms from miners
/:cl:
</details>

Takes away sci comms from miners

Idrc personally but tipy has been yelling about this all day in dev public and i want them to shut up

### CODE REVIEW

**Comment by Tipy1802** [comment]
Love you <3

**Comment by N3D6** [comment]
This is just a horribly shitty idea. Lets take away the only reliable way for miners to speak to the people they need for their tech and upgrades.

**Comment by alexkar598** [comment]
i mean 
image

**Comment by redmoogle** [comment]
Instead of just removing the problem you end up amplifying it, good job. because now they'll more that likely start breaking in because who the fuck reads common? certainly not scientists.

**Comment by Tipy1802** [comment]
Miners can go in person to ask, use the message console, use the common radio, use pda message, tell the AI to mention it to science and many other ways to communicate to science that are less toxic than constant manchildren throwing a hissy fit because they got their tech a few minutes later.

Having miners yell at you for not being mega optimal on rushing their precious tech and constantly back sit gaming you is the worst part of playing science and the most toxic interaction in the whole game

The science-Mining interaction is completely one way: miners yelling at science for tech and science never asks mining for anything or do any other communication with them. Every department needs research not just mining so it makes absolutely no sense why miners get special treatment.

Legit having to deal with miners always ruins my mood and it’s horrible

**Comment by Tipy1802** [comment]
> Instead of just removing the problem you end up amplifying it, good job. because now they'll more that likely start breaking in because who the fuck reads common? certainly not scientists.

Except admins can ban miners for that but can’t for speaking on tcomms

**Comment by redmoogle** [comment]
> 
> 
> > Instead of just removing the problem you end up amplifying it, good job. because now they'll more that likely start breaking in because who the fuck reads common? certainly not scientists.
> 
> Except admins can ban miners for that but can’t for speaking on tcomms

then do it please, but oh wait have any powergaming miners been banned? no.



> 
> 
> Miners can go in person to ask, use the message console, use the common radio, use pda message, tell the AI to mention it to science and many other ways to communicate to science that are less toxic than constant manchildren throwing a hissy fit because they got their tech a few minutes later.
> 
> Having miners yell at you for not being mega optimal on rushing their precious tech and constantly back sit gaming you is the worst part of playing science and the most toxic interaction in the whole game
> 
> The science-Mining interaction is completely one way: miners yelling at science for tech and science never asks mining for anything or do any other communication with them. Every department needs research not just mining so it makes absolutely no sense why miners get special treatment.
> 
> Legit having to deal with miners always ruins my mood and it’s horrible

They'll throw a hissy fit in common and probably ooc afterround

**Comment by UselessTheremin** [comment]
tipy play miner for ~50 hours and come back with your opinion i dont want you spouting these kinda baseless claims

**Comment by Tipy1802** [comment]
And how are they baseless exactly?
I am literally describing my experience which is indeed a valid basis

**Comment by LazennG** [comment]
I'm gonna stop joking about bombing rnd and actually do it

**Comment by athena413** [comment]
> > Instead of just removing the problem you end up amplifying it, good job. because now they'll more that likely start breaking in because who the fuck reads common? certainly not scientists.
> 
> Except admins can ban miners for that but can’t for speaking on tcomms

you are an admin

**Comment by UselessTheremin** [comment]
> And how are they baseless exactly?
> I am literally describing my experience which is indeed a valid basis

not experience as a miner nerd before taking something away you should understand its use properly

**Comment by Mesalikepie** [comment]
No. It is going to increase grief between science and mining needlessly. Additionally robotics use the comms to communicate about mining mechs, and common does not work effectively. Scientists are commonly newer players, due to it being a low skill role, leading to them not reading chat as effectively. Leaving it on science comms helps a lot reduce grief, and it does not need to be removed.

**Comment by Tipy1802** [comment]
> No. It is going to increase grief between science and mining needlessly. 

Every other round when I play scientist I am moments away from bombing the hell out of miners using LAM because of miner’s behaviour on comms (something that admins cannot ban miners for). The possibility of me snapping and griefing mining disappears with the removal of their access to the science comms.
If miners start griefing science at least we (admins) can finally ban them so the griefing will be short lived.

> Additionally robotics use the comms to communicate about mining mechs,

This almost never happens...
Every time I have played robotics and actually asked miners if they want a mech they did not respond, because most miners don’t actually care about listening to the science channel, they only use it to yell at science for upgrades and do not listen to science’s requests as miners use the supply channel as their main communication channel. There are times I have made the mech and miners didn’t end up using it. I have also been told they are not even that good. 
And why should miners be privileged with special access to the science comms and not other jobs as well? Engineers could use a construction mech, atmos techcies a firefighter mech and medics an Odysseus, yet they don’t get access to the science comms...

> and common does not work effectively

Well then use any of the other communication methods! Send them a PDA message, they are hard to not notice. Use the communication console and make a request! You know the consoles on the walls that are designed for signing requests to other departments and broadcast the request to the desired departmental radio channel. Or you know just go in person, robotics has a public lobby for a reason.

**Comment by LazennG** [comment]
> > No. It is going to increase grief between science and mining needlessly.
> 
> Every other round when I play scientist I am moments away from bombing the hell out of miners using LAM because of miner’s behaviour on comms (something that admins cannot ban miners for). The possibility of me snapping and griefing mining disappears with the removal of their access to the science comms.
> If miners start griefing science at least we (admins) can finally ban them so the griefing will be short lived.
> 
> > Additionally robotics use the comms to communicate about mining mechs,
> 
> This almost never happens...
> Every time I have played robotics and actually asked miners if they want a mech they did not respond, because most miners don’t actually care about listening to the science channel, they only use it to yell at science for upgrades and do not listen to science’s requests as miners use the supply channel as their main communication channel. There are times I have made the mech and miners didn’t end up using it. I have also been told they are not even that good.
> And why should miners be privileged with special access to the science comms and not other jobs as well? Engineers could use a construction mech, atmos techcies a firefighter mech and medics an Odysseus, yet they don’t get access to the science comms...
> 
> > and common does not work effectively
> 
> Well then use any of the other communication methods! Send them a PDA message, they are hard to not notice. Use the communication console and make a request! You know the consoles on the walls that are designed for signing requests to other departments and broadcast the request to the desired departmental radio channel. Or you know just go in person, robotics has a public lobby for a reason.

3415cdd0b993273b877f62eda19ec054

**Comment by enkidienne** [comment]
as someone with 200 hours + as science roles, I can safely say that this is NOT a problem... at all. Like maybe 5% of the rounds will have bad interaction between science and mining.
